backend/telemetry_core/src/aggregator/inner_loop.rs

```diff
@@ -55,13 +55,13 @@ pub enum FromShardWebsocket {
     Add {
         local_id: ShardNodeId,
         ip: std::net::IpAddr,
-        node: common::node_types::NodeDetails,
+        node: Box<common::node_types::NodeDetails>,
         genesis_hash: common::node_types::BlockHash,
     },
     /// Update/pass through details about a node.
     Update {
         local_id: ShardNodeId,
-        payload: node_message::Payload,
+        payload: Box<node_message::Payload>,
```

Comment:
`std::mem::size_of::<Payload>() == 344`,  I'm still in favor to avoid to box this but not sure where to cut the threshold without benching it ^^.

Clippy seems to use 200 as default, https://rust-lang.github.io/rust-clippy/master/#large_enum_variant